phpunit.xml

```diff
@@ -1,25 +1,25 @@
 <?xml version="1.0" encoding="utf-8" ?>
-<phpunit bootstrap="bootstrap.php"
+<phpunit bootstrap="tests/unit/bootstrap.php"
```

Comment:
@DeepDiver1975 - you used `failOnWarning` recently - should we add it directly here?

Reply:
I have added it below. Let's see if CI passes.